src/database/seeds/4-create-users.seed.ts

```diff
@@ -11,7 +11,7 @@ export default class CreateUsers implements Seeder {
       .into(User)
       .values([
         {
-          id: 'user',
+          id: '100919359573998466075',
```

Comment:
Place a comment, so we at least see who has this id.

```suggestion
          id: '100919359573998466075', // Felipe
```